cmd/qperf/qperf.cpp

```diff
@@ -123,7 +130,11 @@ namespace {
             return std::shared_ptr<PerfSubscribeTrackHandler>(new PerfSubscribeTrackHandler(full_track_name));
         }
 
-        void ObjectReceived(const quicr::ObjectHeaders&, quicr::BytesSpan) override {}
+        void ObjectReceived(const quicr::ObjectHeaders&, quicr::BytesSpan data) override
+        {
+            metrics_.objects_received++;
+            metrics_.bytes_received += data.size();
```

Comment:
I don't love this, but for some reason on my machine the GetMetrics for subscribers was reporting 0s.

Reply:
It's because upon callback in transport.cpp, it's not updated. It needs to be updated under `ProcessStreamDataMessage()` and `OnRecvDgram()`